### DISCUSSION TITLE
Is There a Fluree API?

### DISCUSSION THREAD

**Discussion by mdebellis** [fluree/db#730] [Q&A]
Most graph databases support APIs in Python and Java. So if you want to programmatically iterate over the instances of a class and get specific property values, execute SPARQL queries from programs, create new instances, classes, properties, utilize libraries like Spacy, QT, NLTK, Jupyter, you can do that. Is there a Python or Java API for Fluree?

**Comment by frydj** [comment]
Check out the guide on the Fluree HTTP API here --> https://developers.flur.ee/docs/reference/http-api/; this will show the basics of querying/transacting using Fluree's API. You can also find lots of good examples in the cookbook (https://developers.flur.ee/docs/reference/cookbook/)
